src/Javascript/Main.js

```diff
@@ -335,17 +336,18 @@ async function linkApp({
     const pathExists = await fs.exists(path)
 
     if (!pathExists) {
-      if (!canPermissionFiles || path.endsWith("/")) {
+      if (!canPermissionFiles || wn.path.isDirectory(path)) {
         await fs.mkdir(path, { localOnly: true })
       } else {
         await fs.write(path, "", { localOnly: true })
       }
       madeFsChanges = true
     }
 
+    const posixPath = wn.path.toPosix(path)
     const adjustedPath = canPermissionFiles
```

Comment:
You might need to make this an absolute path for backwards compatibility. The new SDK supports both absolute and relative, and parses it into the same list format.